README.md

```diff
@@ -72,7 +72,14 @@ Download the latest binary executable for your operating system:
   dnf install tektoncd-cli
   ```
 
-If you have [go](https://golang.org/) installed, `GO111MODULE="on" go get github.com/tektoncd/cli@v0.5.0` is all you need!
+* Source install
+   If you have [go](https://golang.org/) installed and you want to compile the CLI from source you can checkout the [Git repository](https://github.com/tektoncd/cli) and run the following commands:
+
+```shell
+export GO111MODULE=on
```

Comment:
We could add a comment that it might not needed in some case (if you are using go 1.13+ and not in `GOPATH`), but meh :stuck_out_tongue:

Reply:
but meh indeed 🤣